lib/FeedItem.php

```diff
@@ -428,6 +428,37 @@ public function setUid($uid) {
 		return $this;
 	}
 
+	/**
+	 * Get duration (for items where the link points to media)
+	 *
+	 * Use {@see FeedItem::setDuration()} to set the duration.
+	 *
+	 * @return int Duration in seconds.
+	 */
+	public function getDuration() {
```

Comment:
Might be nice to use strict typing:

```suggestion
	public function getDuration(): ?int {
```